test/src/tests-batchUploader.ts

```diff
@@ -862,7 +863,7 @@ describe('batch uploader', () => {
 
             // First upload is successful
             window.fetchMock.post(urls.eventsV3, 200, {
-                overwriteRoutes: false,
+                overwriteRoutes: true,
```

Comment:
Why is this being changed to true?

Reply:
Weird.  for some reason it was failing when on true so i switched it...but seems to be working again.  I'll revert